Handle request send hotkey for gRPC

vincendep opened the pull request:
<!--
Please open an [Issue](https://github.com/kong/insomnia/issues/new) first to discuss new
features or non-trivial changes. Please provide as much detail as possible on the change as
possible including general description, implementation details, potential shortcommings, etc.

If this PR closes an issue, please mention "Closes #XX" where #XX is the issue number.
-->

Closes #3272 
changelog(Fixes): The “Send Request” hotkey now works for gRPC requests

develohpanda commented on packages/insomnia-app/app/ui/components/panes/grpc-request-pane/index.tsx:
```diff
@@ -69,93 +73,102 @@ export const GrpcRequestPane: FunctionComponent<Props> = ({
+  };
+
+  return (
+    <KeydownBinder onKeydown={_handleKeyDown}>
```
We'll want to memoize this arrow function using `useCallback`, otherwise it will change every time this component updates causing unnecessary re-renders.

Ideally this `onKeyDown` handler is actually created within `useActionHandlers` but that involves a bit more of a refactor to account for the conditions, so we can just memoize here for the time being.

develohpanda reviewed (changes requested):
I've just made a note to memoize the handler, but the feature works well!

develohpanda committed:
reduce dependency array surface area

develohpanda reviewed (approved):
Tested and LGTM, thank you for the PR! 🎉

Commit:
Add KeyBoardBinder to GrpcRequestPane

Catch REQUEST_SEND hotkey in GrpcRequestPane and handle it

Commit:
useCallback for key down handler

dimitropoulos committed:
update variable name to be more consistent

and to be clear: more consistent with the rest of the app and where we're headed.  I can see that the rest of the `executeHotKey` calls use `e`.  I'll put up a separate PR for that.

dimitropoulos committed:
removes unnecessary newlines

dimitropoulos reviewed (approved):
thank you so much for the PR.  despite being small, this is a really important feature.  I personally apologize for it taking so long for this to get over the finish line.  we really do appreciate the contribution!

vincendep commented:
Hi @dimitropoulos, happy to contribute!!
